### PR TITLE
Power budget

### PR DESCRIPTION
**Pull request by RicardoCorpaM**
Power budget method to calculate all the input power for an Icepak design.

Restrictions:
- Power coming from a linked EM design, including a PCB analysis, will not be accounted.

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
@MaxJPRey Please could you help to identify why is IPython UT failing? Locally it is working.

File: _unittest/test_98_Icepak.py [hunk 1]
```diff
@@ -468,3 +471,9 @@ def test_39_update_assignment(self):
         assert bound.update_assignment()
         bound.props["Objects"].remove(box2)
         assert bound.update_assignment()
+
+    def test_40_power_budget(self):
+        self.power_budget = self.local_scratch.copyfile(source_power_budget)
+        app = Icepak(self.power_budget, specified_version=desktop_version)
+        power_boundaries, total_power = app.post.power_budget(temperature=20)
+        assert total_power == 787.5221374239883
```
Comment (MaxJPRey) [inline _unittest/test_98_Icepak.py:479]:
```suggestion
        assert abs(total_power - 787.5221374239883) <1
```

File: _unittest/test_98_Icepak.py [hunk 2]
```diff
@@ -468,3 +471,9 @@ def test_39_update_assignment(self):
         assert bound.update_assignment()
         bound.props["Objects"].remove(box2)
         assert bound.update_assignment()
+
+    def test_40_power_budget(self):
+        self.power_budget = self.local_scratch.copyfile(source_power_budget)
+        app = Icepak(self.power_budget, specified_version=desktop_version)
+        power_boundaries, total_power = app.post.power_budget(temperature=20)
+        assert abs(total_power - 787.5221374239883) <1
```
Comment (MaxJPRey) [inline _unittest/test_98_Icepak.py:479]:
```suggestion
        assert abs(total_power - 787.5221374239883) < 1
```

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Pull request merged**